src/tools/interop/idt/idt.py

```diff
@@ -0,0 +1,237 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import argparse
+import logging
+import os
+import shutil
+import sys
+from pathlib import Path
+
+from capture import CaptureEcosystems, CapturePlatforms, PacketCaptureRunner
+from capture.file_utils import border_print, create_file_timestamp, safe_mkdir
+from discovery.matter_ble import MatterBleScanner
+from discovery.matter_mdns import MatterMdnsListener
+
+# TODO argument for log level
+logging.basicConfig(
+    format='%(asctime)s.%(msecs)03d %(levelname)s {%(module)s} [%(funcName)s]\n%(message)s \n',
+    level=logging.INFO)
+
+
+class InteropDebuggingTool:
```

Comment:
Can we require that the tool is run on Rasperry Pi with a specific base image?

Reply:
We could check some system properties to do a good enough job I'm sure, but I'm hoping this can run on as many platforms as easily as possible. The Pi is primarily meant to be a solution that controls as many variables as possible for widest availability / ease of setup.